jquery.pickadate/jquery.pickadate.d.ts

```diff
@@ -99,221 +60,303 @@ interface pickadateOptions extends pickerOptions {
-    get(thing: 'open'): boolean;
+        /** Returns a unique 9-digit integer that is the ID of the picker. */
+        get(thing: 'id'): number;
 
-    /** Returns a boolean value of whether the picker has started or not. */
-    get(thing: 'start'): boolean;
+        /** Returns an array of items that determine which item objects to disable on the picker. */
+        get(thing: 'disable'): any[];
 
-    /** Returns a unique 9-digit integer that is the ID of the picker. */
-    get(thing: 'id'): number;
+        /** Returns a formatted string for the item object specified by `thing` */
+        get(thing: string, format: string): string;
 
-    /** Returns an array of items that determine which item objects to disable on the picker. */
-    get(thing: 'disable'): any[];
+        /** Set the properties, objects, and states to change the state of the picker. */
+        set(thing: string, value?: any, options?: SetOptions): TPickerObject;
+        set(things: TOptions, options?: SetOptions): TPickerObject;
 
-    /** Returns a formatted string for the item object specified by `thing` */
-    get(thing: string, format: string): string;
+        /** Bind callbacks to get fired off when the relative picker method is called. */
+        on(methodName: string, callback: (data?: any) => void): TPickerObject;
 
-    /** Set the properties, objects, and states to change the state of the picker. */
-    set(thing: string, value?: any, options?: any): TPickerObject;
-    set(things: TOptions, options?: any): TPickerObject;
+        /** Bind multiple callbacks at once to get fired off when the relative picker method is called. */
+        on(callbackObject: CallbackObject): TPickerObject;
 
-    /** Bind callbacks to get fired off when the relative picker method is called. */
-    on(methodName, callback: () => void): TPickerObject;
+        /** Unbind callbacks that are bound using the on method. */
+        off(...methodName: string[]): TPickerObject;
 
-    /** Bind multiple callbacks at once to get fired off when the relative picker method is called. */
-    on(callbackObject: CallbackObject): TPickerObject;
+        /** Trigger callbacks that have been queued up using the the on method. */
+        trigger(event: string, data?: any): TPickerObject;
+    }
 
-    /** Trigger callbacks that have been queued up using the the on method. */
-    trigger(event: string): TPickerObject;
+    interface DatePicker extends Picker<DatePicker, DateItem, SetObject> { }
+    interface TimePicker extends Picker<TimePicker, TimeItem, TimeSetObject> { }
 }
 
-interface DatePickerObject extends PickerObject<DatePickerObject, DatePickerItemObject, SetThings> { }
-interface TimePickerObject extends PickerObject<TimePickerObject, TimePickerItemObject, TimePickerSetThings> { }
-
 interface JQuery {
-    pickadate(methodName: "picker"): DatePickerObject;
+    pickadate(methodName: "picker"): Pickadate.DatePicker;
     pickadate(methodName: string): any;
```

Comment:
should these really return `any`? I have more detailed typings for this interface at the bottom of `pickadate.d.ts` as well as JSDoc comments. consider bringing those over here.